Gifski/AppDelegate.swift

```diff
@@ -297,16 +280,30 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
 		circularProgress.progress = 0
 		circularProgress.animated = true
 
-		progress = Progress(totalUnitCount: 1)
-		progress?.becomeCurrent(withPendingUnitCount: 1)
-		gifski = Gifski.convertFile(
-			inputUrl,
-			outputUrl: outputUrl,
-			quality: defaults["outputQuality"] as! Double,
-			dimensions: choosenDimensions,
-			frameRate: choosenFrameRate
-		)
-		progress?.resignCurrent()
+		let progress = Progress(totalUnitCount: 1)
+
+		progress.performAsCurrent(withPendingUnitCount: 1) {
+			let conversion = Conversion(
```

Comment:
I think it should be `Gifski.Conversion`, so it's scoped under the `Gifski` name.

Reply:
Agree 👍

Reply:
Should I move GifskiConversionError to Gifski.Error as well?

Reply:
> Should I move GifskiConversionError to Gifski.Error as well?

👍